### PR TITLE
stringify remote offer id, change offer locale default

### PR DESCRIPTION
**Pull request by renius**

### CODE REVIEW

**Comment by st0ck** [comment]
:+1:

File: db/migrate/20151118121511_add_name_to_offer.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddNameToOffer < ActiveRecord::Migration
+  def change
+    add_column :core_offers, :name, :string, default: ''
```
Comment (ri-nat) [inline db/migrate/20151118121511_add_name_to_offer.rb]: `null: false`

**Comment by idoo** [comment]
:sunglasses:

**Comment by renius** [comment]
@idoo ?

**Comment by idoo** [comment]
:+1:

File: db/migrate/20151109101825_change_offers_remote_offer_id.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class ChangeOffersRemoteOfferId < ActiveRecord::Migration
+  def up
+    change_column :core_offers, :remote_offer_id, :string
```
Comment (ri-nat) [inline db/migrate/20151109101825_change_offers_remote_offer_id.rb]: `default: '', null: false` :)

**Pull request merged**